### DISCUSSION TITLE
Require two reviews for merge

### DISCUSSION THREAD

**Discussion by Rumperuu** [markcheret/footnotes#126] [Ideas]
Currently, PRs only require one passing review before they can be merged. I think we should increase this to two.

For the time being, whilst the team is just us three, this ensures that everyone is aware of, and is happy with, whatever is going into the codebase.

It also ensures that every proposed change is reviewed and approved from both a project direction (i.e., @markcheret) and a coding (i.e., @lolzim and me) perspective.

**Comment by lolzim** [comment]
OK for me, just keep in mind that we all can be on weekends, holydays & the like, so getting the 2nd could take time.

**Comment by markcheret** [comment]
I like the way of thinking.
@lolzim has a good point with taking longer time. I gave it some thought and in the interest of thorough reviews, I'd rather have code that does what it should than a nervous finger on the release trigger.

**Comment by markcheret** [comment]
Oh, and I'm changing the settings for the main branch now.
